explainers/MultiViewport/explainer.md

```diff
@@ -0,0 +1,112 @@
+
+## Usage
+
+### Add a new viewport
+
+Users will be able to add a new device to the viewport by simply checking a box from the dropdown list of devices in the viewport toolbar. Once a device is added, it will appear either next to or underneath the existing viewport(s) with a toolbar on top of it. The devices will be re-centered to accommodate the new viewport. As more devices are added and the viewport fills up, the user will be able to vertically scroll down the viewport to look at all the devices.
+
+### Open DevTools against a new viewport
+
+DevTools will remain the same, regardless of the number of devices in the viewport. As in the current version of DevTools the user will be able to choose the placement of DevTools on their screen and expand/shrink the DevTools window.
+
+### Synchronize browsing and inspection across viewports
+
+Browsing will be synchronized across devices in the viewport, meaning that scrolling in one device would mean all other devices would be kept in sync so they are displaying as close to the same content as possible.. The same will also be true for inspecting and selecting items across different devices in the viewport.
+
+## Design
+
+We have two designs: one for the feature as a whole and one for our intern project implementation. While the implementation of our project will start with only adding a second device, the design we created is scalable to allow for a larger number of devices all in one viewport. The first design adds an entirely new viewport (almost like a new tab) for a second device, which while it works well for two devices, would not be manageable for a larger scope. The second design takes this into account and adds multiple devices into a grid formation in one viewport so users can more easily view a greater number of devices.
+
+Link to design for our project implementation (scoped for two viewports): 
+
+[Design Draft 1](#https://microsoft-my.sharepoint.com/:b:/p/t-hanamemon/Ef3fIWdbAHpLigegfGOgx_IBmWO8e6FayqVmvkjALdJwiw?e=tmLd9C)
+
+[Design Draft 2](#https://microsoft-my.sharepoint.com/:b:/p/t-evelynchou/ESm5uGXiKZ5KmAgAXnTuTlsBO0llV8PjQa5_VkAHZB3aVg?e=OQ6Kgh)
+
+### Open Questions and Ideas
+
+- Zoom percentage (synchronized across devices?).
+- The user’s device is smaller than the screen they want to test (what would the default be?).
+- How many screens does user want to see all at once before they need to scroll (default vs user customization).
+- How would this be affected by placement of sidebar.
+
+## Sizing the Business Opportunity
+
+Given that Device Mode is the second most used feature in DevTools users, such as professional developers and website testers, will continue to rely on Edge DevTools to streamline their processes. This feature will drive developers to Edge when they're specifically testing multiple screen sizes/devices/layouts.
```

Comment:
I'm not sure this business opportunity should be in the explainer document. An explainer document's goal is to detail a user problem/opportunity and a solution for it. It is not meant to explain how it fits in our internal strategy as a team.